website/docs/reference/resource-properties/freshness.md

```diff
@@ -75,11 +73,19 @@ Freshness blocks are applied hierarchically:
 - A `freshness` and `loaded_at_field` property added to a source _table_ will override any properties applied to the source.
 
 This is useful when all of the tables in a source have the same `loaded_at_field`, as is often the case.
+</VersionBlock>
 
 ## loaded_at_field
+
+<VersionBlock firstVersion="1.7">
+(Optional on adapters that support pulling freshness from warehouse metadata tables, required otherwise.)
+</VersionBlock>
+
+<VersionBlock lastVersion="1.6">
 (Required)
 </VersionBlock>
-A column name (or expression) that returns a timestamp indicating freshness.
+
+<br/><br/>A column name (or expression) that returns a timestamp indicating freshness.
```

Comment:
Adding the `<br/><br/>` tags is a little ugly, but it's the only thing I found that would work to preserve the current formatting.

I tried [several things](https://github.com/dbt-labs/docs.getdbt.com/pull/5207/commits) to no avail.

If we don't like them, I'd suggest that we merge this PR as-is and create a follow-up to address them.